manifests/ops-files/add-pcap-agent.yml

```diff
@@ -2,40 +2,40 @@
 - type: replace
   path: /releases/-
   value:
-    name: pcap-server
+    name: pcap
     version: 0+dev.7
-    url: https://github.com/domdom82/pcap-server-release/releases/download/v0.0.1/pcap-server-0.0.1.tgz
+    url: https://github.com/cloudfoundry/pcap-release/releases/download/v0.0.1/pcap-0.0.1.tgz
     sha1: a1655c1992b911fef3ad0b46c5abb8a0217ef0a2
 
 - type: replace
   path: /variables/-
   value:
-    name: pcap_server_mtls_ca
+    name: pcap_agent_mtls_ca
```

Comment:
in the vars-template.yml you are using `pcap_mtls_ca` as the CA name, so this won't work.
however, I like `pcap_mtls_ca` more, so you may as well rename it here. (see other comment in vars-template.yml)